modules/cli/src/main/scala/scala/cli/commands/export0/Export.scala

```diff
@@ -104,17 +104,18 @@ object Export extends ScalaCommand[ExportOptions] {
   override def runCommand(options: ExportOptions, args: RemainingArgs, logger: Logger): Unit = {
     val initialBuildOptions = buildOptionsOrExit(options)
 
-    val output = options.output.getOrElse("dest")
-    val dest   = os.Path(output, os.pwd)
-    if (os.exists(dest)) {
+    val output             = options.output.getOrElse("dest")
+    val dest               = os.Path(output, os.pwd)
+    val shouldExportToJson = options.json.getOrElse(false)
+
+    if (!shouldExportToJson && os.exists(dest)) {
```

Comment:
I think we should also enable users to save JSON to a file. Something like this should work:

```
scala-cli export . --json -o dest/scala-cli.json
scala-cli export . --json -o dest
```